src/Microsoft.AspNetCore.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -4,10 +4,11 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+#if DOTNET5_4
```

Comment:
Actually you still do. What this does is avoid remove and sort thinking `System.Reflection` isn't needed when the file is open in full .NET mode.